client/src/utils/pg/client/package.ts

```diff
@@ -51,6 +51,10 @@ export class PgClientPackage {
         return import("@solana/spl-token");
       case "@solana/web3.js":
         return import("@solana/web3.js");
+      case "@solana/web3.js-tp":
+        return import("@solana/web3.js-tp");
+      case "@solana/webcrypto-ed25519-polyfill":
+        return import("@solana/webcrypto-ed25519-polyfill");
```

Comment:
Can we also add the package names here? https://github.com/solana-playground/solana-playground/blob/749d80851db123428b4c2b831f5362d919a36d3f/client/src/utils/pg/client/package.ts#L5

Reply:
Will do!